package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-azurevirtualmachines",
-    "version": "0.1.1-alpha",
+    "version": "0.1.1",
```

Comment:
Can you try again with Notices.html? Lol there's a decent chunk of changes in package-lock.json since the last release: 
https://github.com/microsoft/vscode-azurevirtualmachines/compare/v0.1.0...master#diff-32607347f8126e6534ebc7ebaec4853dL3